scenario_player/utils/configuration/token.py

```diff
@@ -40,8 +40,9 @@ def __init__(self, loaded_definition: dict, token_info_fpath: pathlib.Path):
         self.dict = loaded_definition.get("token", {})
```

Comment:
I'd rename this from `dict`to `config`

Reply:
Yeah, all those config classes are a mess of several re-re-re-factorings by multiple people piling up on top of each other...